source/shared/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -218,7 +223,13 @@ For the app to work properly with the current version of the Desktop Wallet, mak
 
 #. :ref:`Download <downloads>` the latest version of the Ledger app if you haven't done so already.
 
-#. Connect the Ledger to the computer, and then follow steps 1-3 in :ref:`Install the Ledger app on macOS <install-ledger-app-macos>`. You don't have to reinstall the certificate or enter recovery mode.
+#. In the folder that you extracted the files to, double-click the ``install.bat`` file.
```

Comment:
The .bat file is only for Windows. Instead the user should run install.sh, by running `./install.sh` from a Terminal.